compilerlibs/META.in

```diff
@@ -1,43 +1,45 @@
-version = "[distributed with OCaml]"
+# @configure_input@
+
+version = "@VERSION@"
 description = "compiler-libs support library"
-directory= "+compiler-libs"
 
 package "common" (
   requires = "compiler-libs"
-  version = "[distributed with OCaml]"
+  version = "@VERSION@"
   description = "Common compiler routines"
   archive(byte) = "ocamlcommon.cma"
   archive(native) = "ocamlcommon.cmxa"
 )
 
 package "bytecomp" (
   requires = "compiler-libs.common"
-  version = "[distributed with OCaml]"
+  version = "@VERSION@"
   description = "Bytecode compiler"
   archive(byte) = "ocamlbytecomp.cma"
   archive(native) = "ocamlbytecomp.cmxa"
 )
 
 package "optcomp" (
   requires = "compiler-libs.common"
-  version = "[distributed with OCaml]"
+  version = "@VERSION@"
   description = "Native-code compiler"
   archive(byte) = "ocamloptcomp.cma"
   archive(native) = "ocamloptcomp.cmxa"
   exists_if = "ocamloptcomp.cma"
 )
 
 package "toplevel" (
-  requires = "compiler-libs.bytecomp"
-  version = "[distributed with OCaml]"
+  requires(byte) = "compiler-libs.bytecomp"
```

Comment:
This changes seem to hit a limitation of dune: the byte and native predicate are not supported on the requires field leading to "unavailable module" errors.

Reply:
Ah, perhaps we arrive at the reason this ended up being separate libraries! In which case, I've pulled that commit - changing that would need to be co-ordinated with Dune gaining the required support.